xds/src/main/java/io/grpc/xds/LookasideChannelLb.java

```diff
@@ -91,19 +64,18 @@ public void handleNameResolutionError(Status error) {
   @Override
   public void shutdown() {
     lrsClient.stopLoadReporting();
-    xdsComms2.shutdownLbRpc();
-    lbChannel.shutdown();
+    xdsClient.shutdown();
   }
 
-  private static final class AdsStreamCallbackImpl implements AdsStreamCallback {
+  private static final class EndpointWatcherImpl implements EndpointWatcher {
 
     final LookasideChannelCallback lookasideChannelCallback;
```

Comment:
Make `EndpointWatcherImpl` non-static and move this fields out.

Reply:
`EndpointWatcherImpl` could be further moved to `LookasideLb` afterwards, and `LookasideChannelLb` may also be removed. This refactoring PR just tries to keep the diff as easy to read as possible, without restructuring things unless necessary.